documentation/Compatibility.md

```diff
@@ -5,9 +5,11 @@
 |Min package version| Max package version |Supported Visual Studio version|
 |----------|-----------|:-------------:|
 |1.0.0|1.0.x|Visual Studio 2017 version 15.7 or higher|
+|1.0.0|1.0.x|Visual Studio 2019 version 16.0 or higher|
```

Comment:
@dtchepak any better way to rephrase this one so as we won't need to add a separate entry for every VS major release?

Reply:
Not sure. Here are a few ideas, not sure if they are better though:

* Visual Studio 2017 version 15.7 or higher, or later release of Visual Studio
* Visual Studio releases from Visual Studio 2017 version 15.7 or later
* Any Visual Studio version from Visual Studio 2017 version 15.7.
* Visual Studio [2017 version 15.7, ]

(last one is my attempt to use [NuGet-style version range](https://docs.microsoft.com/en-us/nuget/reference/package-versioning#version-ranges-and-wildcards), I don't think it's very obvious though.)

Another option is to leave as it is and wait for the third major release before rewording this, it might make it more obvious if we can have something like "Visual Studio (VS 2017 from version 15.7, 2019, 2021, and later releases)". 🤔

Reply:
Ok, let's leave it as it is. Maybe once the docs are updated we can take updated description https://github.com/MicrosoftDocs/visualstudio-docs/issues/3147#issuecomment-486845110